### PR TITLE
Protocol, procedure and account related tables

### PR DESCRIPTION
**Pull request by labkey-martyp**
#### Rationale
Tables, ETLs, triggers and lookups related to protocols, procedures and accounts.

#### Changes
- Protocol Esig, protocol usage and procedure ETLs
- Protocol description ETL and RTF to HTML trigger
- Protocol category, protocol state, protocol type and questionnaire lookups (TSV)
- Account and department ETLs
- Cost center, cost type and expense class lookups (TSV)
- Add staff table audit columns

### CODE REVIEW

File: nirc_ehr/resources/data/cost_center.tsv [hunk 1]
```diff
@@ -0,0 +1,132 @@
+value	title
```
Comment (labkey-jeckels) [inline nirc_ehr/resources/data/cost_center.tsv]: Is NIRC OK with these values being public?

File: nirc_ehr/resources/queries/ehr/protocol.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+require("ehr/triggers").initScript(this);
+
+EHR.Server.TriggerManager.registerHandlerForQuery(EHR.Server.TriggerManager.Events.BEFORE_INSERT, 'ehr', 'protocol', function (helper, scriptErrors, row, oldRow) {
+
+    if (row.description) {
+        plainTextParts = row.description.split("\\loch\\af0")
+        let plainText = "";
+        for(let i = 0; i < plainTextParts.length; i++)
+        {
+            if (i > 0)
```
Comment (labkey-jeckels) [inline nirc_ehr/resources/queries/ehr/protocol.js:11]: Are you intentionally skipping index 0 here?

**Review by labkey-jeckels — APPROVED** [review]

File: nirc_ehr/resources/data/cost_center.tsv (hunk 1 above)
Comment (labkey-martyp) [reply nirc_ehr/resources/data/cost_center.tsv]: Yeah sounds like they are pretty arbitrary values as they explained it to us, however I was debating if it should ETL or TSV, so converted to ETL just to ensure no issues.

File: nirc_ehr/resources/queries/ehr/protocol.js (hunk 2 above)
Comment (labkey-martyp) [reply nirc_ehr/resources/queries/ehr/protocol.js:11]: Yes the first part is all RTF formatting

**Pull request merged**